lib/new_relic/telemetry/ecto/metadata.ex

```diff
@@ -1,16 +1,48 @@
 defmodule NewRelic.Telemetry.Ecto.Metadata do
   @moduledoc false
 
-  @postgrex_select ~r/FROM "(?<table>\w+)"/
-  @postgrex_insert ~r/INSERT INTO "(?<table>\w+)"/
-  @postgrex_update ~r/UPDATE "(?<table>\w+)"/
-  @postgrex_delete ~r/FROM "(?<table>\w+)"/
-  @postgrex_create_table ~r/CREATE TABLE( IF NOT EXISTS)? "(?<table>\w+)"/
   def parse(%{
         query: query,
         result: {_ok_or_error, %{__struct__: struct}}
       })
       when struct in [Postgrex.Result, Postgrex.Error] do
+    parse_query(query, "Postgres")
+  end
+
+  def parse(%{
+        query: query,
+        result: {_ok_or_error, %{__struct__: struct}}
+      })
+      when struct in [MyXQL.Result, MyXQL.Error] do
+    parse_query(query, "MySQL")
+  end
+
+  def parse(%{
+        query: query,
+        result: {_ok_or_error, %{__struct__: struct}}
+      })
+      when struct in [Exqlite.Result, Exqlite.Error] do
+    parse_query(query, "SQLite3")
+  end
+
+  def parse(%{query: query, repo: repo, result: {_ok_or_error, _}}) do
```

Comment:
This part has not been tested due to several technical difficulties:
- Docker images of Oracle instances require license agreement.
- jamdb_oracle does [not support](https://github.com/erlangbureau/jamdb_oracle/blob/d5c87dd04fdf2206d27f09a13144a115764ee429/lib/jamdb_oracle_ecto.ex#L189-L193) storage_down storage_up.
- The volume size of an Oracle instance is about 3 gigabytes. If we test every time, it takes a lot of time to initialize.